mixer/pkg/runtime/dispatcher.go

```diff
@@ -220,7 +220,7 @@ func (m *dispatcher) Check(ctx context.Context, requestBag attribute.Bag) (*adap
 		},
 	)
 	res, _ := cres.(*adapter.CheckResult)
-	if glog.V(3) {
+	if res != nil && glog.V(3) {
 		glog.Infof("Check %s", res)
```

Comment:
If we change this to `glog.Infof("Check %v", res)`, do we still panic ?

Reply:
This seems ok. Alternatively (or in parallel), we could change the code in `status.String` to handle `nil` better.

Reply:
@douglas-reid The crash is because `res` is nil. 
@guptasu We probably don't want to log `Check nil` from glog.Inof("...%v", ...)

Reply:
Can you check why the response was nil?
I think `%v` is better because it tells us that the response was `nil`

Reply:
That is fine. I will see if I can get some time to dig deeper